docs/install/kubernetes.md

```diff
@@ -107,180 +125,129 @@ are being pulled locally for the first time (it will take an average of 10-15 mi
+    mlrun-ce/mlrun-ce
 ```
-Where `$(minikube ip)` shell command resolving the external node address of the k8s node VM.
 
-## Installing Kubeflow
+## Configuring Online Feature Store
+MLRun Community Edition now supports online feature store. To enable it, you need to first deploy a REDIS service which should be accessible to your MLRun CE cluster.
+To deploy a REDIS service please refer to the following [link](https://redis.io/docs/getting-started/).
+
+Once you have a REDIS service deployed, you can configure MLRun CE to use it by adding the following helm value configuration to your helm install command:
 
-You can run your functions while saving outputs and artifacts in a way that is visible to Kubeflow Pipelines.
-To use this capability you need to install Kubeflow on your cluster.
-Refer to the [Kubeflow documentation](https://www.kubeflow.org/docs/started/getting-started/) for more information.
+```bash
+--set mlrun.api.extraEnvKeyValue.MLRUN_REDIS__URL=<redis-address>
+```
 
 ## Usage
 
 Your applications are now available in your local browser:
+- jupyter-notebook - http://<host-machine-address>:30040
+- nuclio - http://<host-machine-address>:30050
+- mlrun UI - http://<host-machine-address>:30060
+- mlrun API (external) - http://<host-machine-address>:30070
+- minio API - http://<host-machine-address>:30080
+- minio UI - http://<host-machine-address>:30090
+- pipeline UI - http://<host-machine-address>:30100
+- grafana UI - http://<host-machine-address>:30110
 
-- Jupyter-notebook - http://localhost:30040
-- Nuclio - http://localhost:30050
-- MLRun UI - http://localhost:30060
-- MLRun API (external) - http://localhost:30070 (health check via http://localhost:30070/api/healthz)
 
 ```{admonition} Check state
 You can check current state of installation via command `kubectl -n mlrun get pods`, where the main information
 is in columns `Ready` and `State`. If all images have already been pulled locally, typically it will take 
 a minute for all services to start.
 ```
 
-```{admonition} Note
-The above links assume your Kubernetes cluster is exposed on localhost.
-If that's not the case, the different components are available on the provided `externalHostAddress`
-- You can change the ports by providing values to the helm install command.
-- You can add and configure a k8s ingress-controller for better security and control over external access.
-```
+> **Note:**
+> - You can change the ports by providing values to the helm install command.
+> - You can add and configure a k8s ingress-controller for better security and control over external access.
 
-## Start working
 
-Open Jupyter Lab on [**jupyter-lab UI**](http://localhost:30040) and run the code in [**quick-start.ipynb**](https://github.com/mlrun/mlrun/blob/master/docs/quick-start/quick-start.ipynb) notebook.
+## Start Working
+
+Open Jupyter Notebook on [**jupyter-notebook UI**](http://localhost:30040) and run the code in 
```

Comment:
Open the Jupyter notebook on [**jupyter-notebook UI**](http://localhost:30040) and run the code in the 
[**examples/mlrun_basics.ipynb**](https://github.com/mlrun/mlrun/blob/master/examples/mlrun_basics.ipynb) notebook.